contracts/proxy-lib/tests/sandbox.rs

```diff
@@ -123,7 +123,7 @@ async fn test_view_proposal() -> Result<()> {
         .await?;
 
     let view_proposal: Option<Proposal> = proxy_helper
-        .view_proposal(&relayer_account, &proposal_id)
+        .view_proposal(&relayer_account, *proposal_id)
```

Comment:
it could've been this

```suggestion
        .view_proposal(&relayer_account, proposal_id)
```